hypertrace-graphql-gateway-service-metric-utils/src/main/java/org/hypertrace/graphql/utils/metrics/gateway/MetricBaselineAggregationConverter.java

```diff
@@ -0,0 +1,51 @@
+package org.hypertrace.graphql.utils.metrics.gateway;
+
+import io.reactivex.rxjava3.core.Single;
+import org.hypertrace.core.graphql.common.utils.Converter;
+import org.hypertrace.gateway.service.v1.baseline.Baseline;
+import org.hypertrace.graphql.metric.schema.Health;
+import org.hypertrace.graphql.metric.schema.MetricBaselineAggregation;
+
+import java.util.Optional;
+
+class MetricBaselineAggregationConverter
+    implements Converter<Optional<Baseline>, MetricBaselineAggregation> {
+
+  @Override
+  public Single<MetricBaselineAggregation> convert(Optional<Baseline> baseline) {
```

Comment:
Generally we don't want to accept optionals as an arg - the caller shouldn't call the function if it doesn't have a baseline to convert. Please remove the default valueing happening below and null out the whole baseline object if we don't have a value to convert.